pysilcam/tracking/track.py

```diff
@@ -0,0 +1,417 @@
+from docopt import docopt
+from pysilcam.tracking.silcamtracker import *
+import numpy as np
+import sys
+import glob
+import pandas as pd
+from scipy import interpolate
+import matplotlib.pyplot as plt
+from skimage.transform import rotate
+from pysilcam.postprocess import explode_contrast
+from pysilcam.config import PySilcamSettings, settings_from_h5
+
+
+def make_output_path(datapath):
+    dataset_name = os.path.split(datapath)[1]
+    outputpath = os.path.join(datapath, ('../output_' + dataset_name))
+    os.makedirs(outputpath, exist_ok=True)
```

Comment:
F405: 'os' may be undefined, or defined from star imports: pysilcam.tracking.silcamtracker  <!-- Automatically posted by Lintly -->